src/test/java/io/vertx/test/core/AsyncTestBase.java

```diff
@@ -596,19 +596,29 @@ protected void waitUntil(BooleanSupplier supplier) {
     waitUntil(supplier, 10000);
   }
 
+  protected void waitUntilNoFail(BooleanSupplier supplier) {
+    waitUntilNoFail(supplier, 10000);
+  }
+
   protected void waitUntil(BooleanSupplier supplier, long timeout) {
```

Comment:
we should rename this method rather `assertWaitUntil` and make it fail the test instead. The `waitUntilNoFail` should be then just named `waitUntil`.